test/thermo/RedlichKwongMFTP_Test.cpp

```diff
@@ -9,8 +9,9 @@ namespace Cantera
 class RedlichKwongMFTP_Test : public testing::Test
 {
 public:
-    RedlichKwongMFTP_Test() {
-        test_phase.reset(newPhase("../data/co2_RK_example.cti"));
+    void initializeTestPhase(const std::string & filename)
+    {
+        test_phase.reset(newPhase(filename));
```

Comment:
Leaving the constructor as it was would allow you to eliminate the boilerplate code in each test for constructing the `ThermoPhase` object.

Reply:
So just to be clear, if I add  the `test_phase.reset(newPhase...)` back in , I can remove the function `initializeTestPhase()`  and then also the lines 
    const std::string valid_file("../data/co2_RK_example.cti");
    initializeTestPhase(valid_file);
at the beginning of each test?